source/app/service-providers/translations/index.ts

```diff
@@ -105,6 +105,20 @@ export default class TranslationProvider extends ProviderContract {
     global.i18nFallbackRawData = await fs.readFile(fallback.path, 'utf8')
     global.i18nFallback = JSON.parse(global.i18nFallbackRawData)
 
+    // Get user's setting of translation updates
+    const checkForTranslationUpdates: boolean = this._config.get('system.checkForTranslationUpdates')
+    if (checkForTranslationUpdates) {
+      await this.update()
+    } else {
+      this._logger.info('[Translation Provider] User decide not to update translations.')
+    }
+  }
+
+  /**
+   * Get translation updates via translation server
+   * @return {Promise} Resolves if everything worked out, rejects otherwise.
+   */
+  async update (): Promise<void> {
```

Comment:
I think it might make sense to call this `updateTranslations` to make it explicit that what this function does is solely to download updated translations